### ISSUE TITLE
What is the training details on CUB-200？

### ISSUE THREAD

**Issue by DolphinAC** [recasens/Saliency-Sampler#2]
I find the train loss can not descend on CUB-200.
Could you tell me the training details ,such as learning rate on taskNet and saliencyNet， pretrained dataset，epochs and so on.
